v1/metrics.yaml

```diff
@@ -232,6 +232,59 @@ paths:
               uri: '{{options.host}}/pageviews/top/{project}/{access}/{year}/{month}/{day}'
       x-monitor: false
 
+  /unique-devices/{project}/{access-site}/{granularity}/{start}/{end}:
```

Comment:
If we wanted to be in line with other endpoints, I'd say that this should be named `/metrics/pageviews/unique/{other_parameters}`. However, taking into account the https://phabricator.wikimedia.org/T126947 bug, I understand why this doesn't include the `pageviews` word. 

However, as I understand we want to gradually migrate `/metrics/pageviews/` to `/metrics/views` or something similar? May be we should expose the new endpoint in line with the intended future name for other, older endpoints? So, finish bike shedding on migration first and use a new layout right away?